### PR TITLE
Fixes an error when the title is set to none

### PR DESCRIPTION
**Pull request by jbrazio**
If the Github user has no real name set then an error in throw and notifications do not appear.
This fixes the error by using the git username if none is found.

### CODE REVIEW

File: githubnotifier.py [hunk 1]
```diff
@@ -433,6 +433,10 @@ def update_feeds(self, feeds):
                  'message': message,
                  'icon': user['avatar_path']}
 
+            # If the user has no real name defined, fallback to username
+            if not n['title']:
+                n['title'] =  user.get('login', user['login'])
```
Comment (abiczo) [inline githubnotifier.py]: Did you mean `user.get('login')`?

Passing in `user['login']` as the default seems problematic (ie if 'login' is not present you'd still try to get that key from the user dict)

**Comment by abiczo** [comment]
Sorry about the late response and thanks for fixing this! Just one comment inline before merging your diff.

**Comment by jbrazio** [comment]
This was a quick and dirty hack, I'll update the PR.

**Comment by jbrazio** [comment]
@abiczo I've rebased and included your feedback.

**Comment by abiczo** [comment]
Awesome, thanks!

**Pull request merged**